docs/b2b-edition/specs/storefront/storefront/company.yaml

```diff
@@ -4038,985 +913,933 @@ paths:
+      parameters: []
       responses:
-        '200':
+        "200":
           description: OK
           content:
             application/json:
               schema:
-                description: ''
-                type: object
-                properties:
-                  code:
-                    type: number
-                  message:
-                    type: string
-                    minLength: 1
-                  data:
-                    type: object
-                    properties:
-                      addressId:
-                        type: string
-                        minLength: 1
-                    required:
-                      - addressId
-                required:
-                  - code
-                  - message
-                  - data
-                x-examples:
-                  example-1:
-                    code: 200
-                    message: Success
-                    data:
-                      addressId: '1415'
-              examples:
-                example-1:
-                  value:
-                    code: 200
-                    message: Success
-                    data:
-                      addressId: '1415'
-      description: |-
-        Mark an address as active or inactive.
-        Equivalent Storefront GraphQL API Mutation: `addressDelete`. For more information, see the [GraphQL Playground](https://api-b2b.bigcommerce.com/graphql/playground).
-      tags:
-        - Company
-  '/companies/{companyId}/default-addresses':
+                allOf:
+                  - $ref: "#/components/schemas/responseObject"
+                  - type: object
+                  - properties:
+                      code:
+                        type: number
+                        default: 200
+                      data:
+                        type: object
+                        properties:
+                          addressId:
+                            $ref: "#/components/schemas/addressId"
+  /companies/{companyId}/default-addresses:
     parameters:
-      - schema:
-          type: string
-        name: companyId
-        in: path
-        required: true
-        description: Unique company ID
+      - $ref: "#/components/parameters/companyId"
     get:
-      summary: Get Company Default Addresses
+      tags:
+        - Addresses
+      summary: "Get Company Default Addresses"
+      operationId: get-companies-companyId-default-addresses
+      description: |-
+        Returns a Company’s default shipping and billing addresses. Use Update Address Information to change which addresses are set as the defaults.
+
+        Equivalent Storefront GraphQL API Mutation: `defaultBillingAddress` for the billing address and `defaultShippingAddress` for the shipping address. For more information, see the [GraphQL Playground](https://api-b2b.bigcommerce.com/graphql/playground).
```

Comment:
We need to distinguish between GraphQL mutations and queries.  defaultBillingAddress and defaultShippingAddress are queries, not mutations.  

I didn't exhaustively look for other occurrences like this, but the full docs should be checked for other places where this needs to be changed.